src/components/task-step/all-steps.cjsx

```diff
@@ -109,29 +109,29 @@ ExternalUrl = React.createClass
   displayName: 'ExternalUrl'
   mixins: [StepMixin, StepFooterMixin]
   hideContinueButton: -> true
+  getDefaultProps: ->
+    redirectToUrl: (url) -> window.open( url, '_blank')
```

Comment:
Hm, didn't mark this as outdated since the change happened below, but i've made the switch